### PR TITLE
Add info on where PDF2IMG NuGet package doc is in reference guide

### PR DESCRIPTION
**Pull request by datalogics-saharay**
Let's be more descriptive on where to find the PDF2IMG documentation for the .NET interface used by the NuGet package.

### CODE REVIEW

File: PDF2IMG/index.html [hunk 1]
```diff
@@ -20,7 +20,9 @@ <h1>Datalogics PDF2IMG Documentation</h1>
  <h2>PDF2IMG</h2>
  <p>A command-line utility and API that converts PDF files to a variety of image formats, including PNG, JPG, TIFF, BMP, and more. The product is built upon the Adobe PDF Library and uses Adobe technology for unrivaled color management during the PDF conversion process. While PDF2IMG can be run with as few as two command line arguments, many other optional command line arguments are provided so that you can create custom output. PDF2IMG is available on both Windows and Linux platforms, 32 and 64 bit.</p>
  <br>
- <p>PDF2IMG's .NET interface is now available as an x64 Windows NuGet package.</p>
+ <p>PDF2IMG's .NET interface is now available as an x64 Windows NuGet package. Please see the section marked <b>Appendix: API Calls, .NET Interface </b> in the API reference guide for more information on how to use our PDF2IMG NuGet package.
```
Comment (datalogics-robb) [inline PDF2IMG/index.html:23]: Would it make sense to drop a link to the nuget package here?  i.e. https://www.nuget.org/packages/PDF2IMG.LM.NET#readme-body-tab

**Review by datalogics-robb — COMMENTED** [review]
Would a link make sense here?

File: PDF2IMG/index.html (hunk 1 above)
Comment (datalogics-saharay) [reply PDF2IMG/index.html]: Yes, it wouldn't hurt anything and it might help more people discover the package. Added!

**Review by datalogics-robb — APPROVED** [review]

**Review by datalogics-robb — APPROVED** [review]
The changes look good.

**Review by datalogics-staylor — APPROVED** [review]

**Pull request merged**